tests/securetrack_test/test_secure_track_helper_unittest.py

```diff
@@ -503,10 +503,10 @@ def test_03_network_object_subnet_search(self):
     def test_04_get_network_object_by_device_and_object_id(self):
         self.mock_get_uri.return_value.content = fake_request_response("network_objects")
         with patch('pytos.common.rest_requests.requests.Request') as mock_get_uri:
-            network_object = self.helper.get_network_object_by_device_and_object_id(158, 3418214)
+            network_object = self.helper.get_network_object_by_device_and_object_id(158, 3433950)
```

Comment:
Where did you get this ID. In general it should not be static anyway, but the logic is that is the ID of the object that is used on the test environment tests run on. I'm not sure what is the ID, but if it was wrong, hod did it not fail till now?

Reply:
I got that ID from test [Mock XML] (https://github.com/Tufin/pytos/blob/master/tests/securetrack_test/resources/testnetworkobjects/network_objects.xml). The original test did not verify that the searched object ID existed, so [this commit] (https://github.com/Tufin/pytos/pull/11/commits/ffca274d2cd29186c146aa2db320cf35d9937e23) made the test start failing, because of an IndexError exception.